// Generated by IcedCoffeeScript 1.6.3-f
(function() {
  var a, b, c;



  c = function() {
    return b();
  };

  b = function() {
    return a();
  };

  a = function() {
    setTimeout(function() {
      throw new Error('Error yay!');
    }, 10);
  };

  process.on('uncaughtException', function(err) {
    console.error(err);
    return process.exit(1);
  });

  c();

}).call(this);
